src/pages/admin/AdminSessionPage.tsx

```diff
@@ -0,0 +1,217 @@
+              <VStack spacing={4} align="stretch">
+                <FormControl isRequired>
+                  <FormLabel>Title</FormLabel>
+                  <Input
+                    value={title}
+                    onChange={(e) => setTitle(e.target.value)}
+                    placeholder="Session title"
+                  />
+                </FormControl>
+
+                <FormControl isRequired>
+                  <FormLabel>Key</FormLabel>
+                  <Input
+                    value={key}
+                    onChange={(e) => setKey(e.target.value)}
+                    placeholder="Attendance key"
+                  />
+                </FormControl>
+
+                <FormControl isRequired>
+                  <FormLabel>Expires</FormLabel>
+                  <DatePicker
+                    selected={expiresDate}
+                    onChange={(date: Date | null) => setExpiresDate(date)}
+                    showTimeSelect
+                    timeFormat="HH:mm"
+                    timeIntervals={5}
+                    timeCaption="time"
+                    dateFormat="MMMM d, yyyy h:mm aa"
+                    minDate={new Date()}
+                    customInput={<Input placeholder="Select date and time" />}
+                    className="chakra-input css-1kp110w"
+                  />
+                </FormControl>
+
+                <Button type="submit" colorScheme="blue" isLoading={loading}>
+                  Create Session
+                </Button>
+              </VStack>
+            </form>
+            {error && (
+              <Text color="red.500" mt={4}>
+                {error}
+              </Text>
+            )}
+          </CardBody>
+        </Card>
+
+        <Card>
+          <CardHeader>
+            <HStack>
+              <Users size={20} />
+              <Text>Active Sessions</Text>
+            </HStack>
+          </CardHeader>
+          <CardBody>
+            {loading ? (
+              <Text>Loading sessions...</Text>
+            ) : activeSessions.length === 0 ? (
+              <Text>No active sessions found.</Text>
+            ) : (
+              <VStack spacing={4} align="stretch">
+                {activeSessions.map((session) => (
+                  <Box
+                    key={session.id}
+                    p={4}
+                    borderWidth="1px"
+                    borderRadius="md"
+                    borderColor={borderColor}
+                    bg={cardBg}
+                  >
+                    <VStack align="stretch" spacing={2}>
+                      <Heading size="sm">{session.title}</Heading>
+                      <HStack>
+                        <Key size={16} />
+                        <Text>{session.key}</Text>
+                      </HStack>
+                      <HStack>
+                        <Clock size={16} />
+                        <Text>
+                          {new Date(session.expires).toLocaleString()}
```

Comment:
try to use functions from localization for both presentation and api calls so refactoring all dates is easier